woocommerce/woocommerce-android
# Store creation: Update list of industries and tracking

> **Note**
> Big PR warning is just because of deletion of most of the previous industry values. Actual code is pretty small.

Part of #8824


### Description


This PR updates:
1. the list of industries being shown (it's much more reduced now, [to match Core](https://github.com/woocommerce/woocommerce/blob/07a2f83f53352bc9c24f929b7dd787c95e91f4aa/plugins/woocommerce/src/Internal/Admin/Onboarding/OnboardingIndustries.php#L28-L69)).
2. the list of commerce platforms (added a few things, [to match Woocommerce.com NUX](https://github.com/Automattic/woocommerce.com/pull/16687))
3. the way chosen industry is now Tracked. There's no category anymore, so only 1 key is used: `industry_slug`. This is to match [the change in iOS](https://github.com/woocommerce/woocommerce-ios/pull/9422).
4. the way chosen industry is now passed to the API call. Same as above, it now only needs to pass a value, with `slug` as the key.

This PR implements all the latest discussion and decision up to April 19th, 2023 (p1681867563742539-slack-team-ghidorah) so it's completed. If anything else comes up, it will be added in a separate PR.

### Testing instructions

1. Start a new site on the app, give it a name,
2. On the industries list, make sure the list is more reduced and matched the new values,
3. On the journey, choose "I'm already selling online",
4. On the platforms list, make sure the list is updated with some new merchants (Amazon, Ecwid, Walmart, etc).
5. On the summary screen, check logcat for tracking similar to: `🔵 Tracked: site_creation_profiler_data, Properties: {"industry_slug":"electronic_computers","user_commerce_journey":"im_already_selling_online","ecommerce_platforms":"walmart","country_code":"US","is_debug":true}`

### Video

[updateditems.webm](https://user-images.githubusercontent.com/266376/232968318-19118b10-c43e-46a4-bd05-d716844a57d7.webm)
